modules/push-notifications/src/push-notifications.proto

```diff
@@ -42,5 +55,7 @@ service PushNotifications {
   rpc SetConfig(SetConfigRequest) returns (SetConfigResponse);
   rpc SetNotificationToken(SetNotificationTokenRequest) returns (SetNotificationTokenResponse);
   rpc GetNotificationTokens(GetNotificationTokensRequest) returns (GetNotificationTokensResponse);
-  rpc SendNotification(SendNotificationRequest) returns (SendNotificationResponse);
+  rpc Send(SendNotificationRequest) returns (SendNotificationResponse);
```

Comment:
Since this was renamed, grpc-sdk needs to be updated and usages from other modules need to be updated as well. This is also a breaking change, which would cause a version bump in conduit. Undo the rename.

Reply:
In that case also rename the following for readability and consistency:
`SendMany` -> `SendManyNotifications` 
`SendToManyDevices` -> `SendNotificationToManyDevices`

Reply:
`ConduitGrpcSdk` should also expose wrappers for both of the above.